### ISSUE TITLE
Multithreading in 4rad doesn't seem to be working

### ISSUE THREAD

**Issue by motorsep** [qbism/q2tools-220#20]
I don't think multithreading works at all in these tools :(
I am running 4rad and it only takes up 12% of CPU, whether I run it without -threads or with -threads cmd arg.

**Comment by qbism** [comment]
Built with VS 2019?  What about cross-built exe from the release?

**Comment by motorsep** [comment]
> Built with VS 2019? What about cross-built exe from the release?

Yep, both built with VS2019 here and your cross-compiled binaries. All work as single-threaded app.

I put a breakpoint on numthreads = 1 in single thread block and it always hit there if I don't specify number of threads. As if it doesn't automatically detect number of threads.

If I specify number of threads, it doesn't hit there, but still runs as single threaded. Not only that - it also only utilizes 12% - 19% of single core/thread.

**Comment by qbism** [comment]
With release binaries, I could not duplicate issue in Windows 10 with 8 cores.  All cores ran at 100% during 4rad.

**Comment by motorsep** [comment]
> With release binaries, I could not duplicate issue in Windows 10 with 8 cores. All cores ran at 100% during 4rad.

Am I seeing "works for me, so it works" approach? Like I mentioned earlier, none of the tools (4bsp/vis/rad) work as multithreaded for me and even as single-threaded they only take up 12% of CPU time. Neither locally compiled, nor provided binaries.

If guess if this issue can't be fixed, you can close it.
